lib/components/ScheduleSelector/components/DailySelector/DailySelector.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, useState, useCallback } from 'react'
+import { DayPicker, TimePicker } from '../index'
+import { Select } from '../../../inputs'
+import {
+  EVERY_DAY,
+  MIDNIGHT,
+  SELECT_ALL,
+  SPECIFIC_DAYS,
+  WEEK_DAYS
+} from '../../SchedualeSelector.consts'
```

Comment:
typo in `ScheduleSelector`

also - let's name the file `scheduleSelectorConsts` to avoid having it with `.`

Reply:
I think that the correct way is to have suffix, similar to `.stories.tsx`, we can have `.consts.ts, .utils.ts, .api.ts`, it is know way to manage files

Reply:
about `.stories` - yes, but we already have `tableConsts` and `tableUtils` for example, and should stay consistent

Reply:
Ok, just for the record , I prefer the suffix style :)